### PR TITLE
Variables fixes to match Figma styles

### PR DESCRIPTION
**Pull request by hugosolar**
## Description
This PR fix some Bulma's default values to match with Figma Design Library
- $fullhd size is now **1248px**
- $column-gap is now **3rem**
- Fix `line-height` on paragraph style classes because it looked tight

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/styles/layout.scss [hunk 1]
```diff
@@ -1,2 +1,5 @@
+$fullhd : 1248px + (2 * $gap);
+$column-gap: 3rem;
```
Comment (brenoferreira) [inline src/styles/layout.scss:2]: Shouldn't you change the `$gap` to be `3rem`?

Comment (hugosolar) [reply src/styles/layout.scss:2]: that was my intention but it doesn't seem to work, then, I found [this](https://bulma.io/documentation/columns/gap/). As far as I understand now, $gap is a value to calculate breakpoints

File: src/styles/typography.scss [hunk 2]
```diff
@@ -39,7 +39,7 @@ $body-font-sizes: (
 
 @mixin body-typography($body-name, $body-font-size) {
   .body#{$body-name} {
-    line-height: 1.3;
+    line-height: $body-font-size * 1.6;
```
Comment (dhruvkb) [inline src/styles/typography.scss]: Just using 1.6 in the line-height would be the same. Line heights expressed purely as [numbers](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height#%3Cnumber%3E) are calculated by multiplying with the font-size. 
```suggestion
    line-height: 1.6;
```

File: tests/unit/typography-test.scss [hunk 3]
```diff
@@ -21,7 +21,7 @@
 
       @include expect {
         .body-standard {
-          line-height: 1.3;
+          line-height: 4.096rem;
```
Comment (dhruvkb) [inline tests/unit/typography-test.scss]: `1.6 * 2.56rem` is the same as `4.096rem`, further reinforcing my point.

**Review by dhruvkb — CHANGES_REQUESTED** [review]
There's a simpler way to fix line-heights.

File: src/styles/typography.scss (hunk 2 above)
Comment (hugosolar) [reply src/styles/typography.scss]: Yup but if we use `rem` will be always the same value and that doesn't work for big font sizes.

**Review by dhruvkb — APPROVED** [review]

**Pull request merged**